Print stdout in Hg log

> English is not my first language, so please forgive my grammatical errors

I added a pre-commit hook to the hg repository that uses eslint to check for syntax errors, and when I commit the code via vscode-hg, if an error occurs with eslint, it commits terminal, but I can't see any hint about the eslint rule in the hg ouput.

I see the following output.
```bash
error Command failed with exit code 1.
abort: pre-commit hook exited with status 1

error Command failed with exit code 1.
abort: pre-commit hook exited with status 1
```

Output of the terminal:
```bash
$ eslint '**/*.js' --cache --fix

/Users/4ark/project/helper/404.js
  9:7  error  'a' is assigned a value but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I think this is because vscode-hg only prints stderr, not stdout. 

This is the internal implementation that I know through the source code:
- https://github.com/mrcrowl/vscode-hg/blob/8661ee183e6d2aea9f6c8a53f5171c86644d50ea/src/hg.ts#L620

Because I see that vscode sees the eslint rule error when doing a git commit. I would expect vscode-hg to have the same behaviour (perhaps an optional feature)

thx!

version:
- vscode: 1.65.2
- vscode-hg: v1.8.0

--- a/src/hg.ts
+++ b/src/hg.ts
@@ -617,7 +617,7 @@ export class Hg {
             }
 
             if (options.logErrors !== false && result.stderr) {
-                this.log(`${result.stderr}\n`);
+                this.log(`${result.stdout}\n${result.stderr}\n`);
             }
 
             return Promise.reject<IExecutionResult<string>>(

